static_src/actions/user_actions.js

```diff
@@ -200,10 +200,25 @@ const userActions = {
 
   clearInviteNotifications() {
     AppDispatcher.handleViewAction({
-      type: userActionTypes.USER_INVITE_STATUS_DISMISSED
+      type: userActionTypes.USER_LIST_NOTICE_DISMISSED
     });
   },
 
+  createUserListNotification(noticeType, description) {
```

Comment:
I think thats a good idea. It currently uses the ES6 syntax for exampling the variable name as the key. Instead of `data`, it does what you are describing with `action`. I think it would be a lot of refactoring to change them all.